go/vt/vitessdriver/driver.go

```diff
@@ -167,6 +172,10 @@ type Configuration struct {
 	//
 	// Default: "vitess"
 	DriverName string `json:"-"`
+
+	// SessionToken is a protobuf encoded vtgatepb.Session represented as base64, which
+	// can be used to distribute a transaction over the wire.
+	SessionToken string
```

Comment:
The test `TestConfigurationToJSON` transforms the `Configuration` struct to a JSON string and compares the string with a constant expected output. Because we are adding the field `SessionToken` to `Configuration`, we must update the assertion of `TestConfigurationToJSON`.

The test is failing in GitHub Actions as observed here: https://github.com/vitessio/vitess/runs/4670869522?check_suite_focus=true

Reply:
Thanks for pointing that out. Fixed now.